nvd3/linePlusBarChart.py

```diff
@@ -53,7 +53,7 @@ class linePlusBarChart(TemplateMixin, NVD3Chart):
 
     .. raw:: html
 
-        <div id="linePlusBarChart"><svg style="height:450px; width:100%"></svg></div>
```

Comment:
Reply "why are you removing the width":
Err, I don't know , I didn't modify the code about width.
Seriously I only modify "linePlusBarChart.py", this should not happen...

At 2015-08-16 21:30:16, "Areski Belaid" <notifications@github.com> wrote:


In nvd3/linePlusBarChart.py:

> @@ -53,7 +53,7 @@ class linePlusBarChart(TemplateMixin, NVD3Chart):
>  
>      .. raw:: html
>  
> -        <div id="linePlusBarChart"><svg style="height:450px; width:100%"></svg></div>


why are you removing the width ?

—
Reply to this email directly or view it on GitHub.